src/app/amsterdam/bestuur-en-organisatie/college-van-burgemeester-en-wethouders/page.tsx

```diff
@@ -0,0 +1,130 @@
+'use client'
+
+import { AspectRatio, Breadcrumb, Card, Grid, Heading, PageHeading, Paragraph } from '@amsterdam/design-system-react'
+import NextImage, { type StaticImageData } from 'next/image'
```

Comment:
Why not use our own Image component?

Reply:
We use `NextImage` on other pages, too. @alimpens Can you explain why?